### DISCUSSION TITLE
how can I perform a "right click" context_click?

### DISCUSSION THREAD

**Discussion by webmoskal** [seleniumbase/SeleniumBase#1277] [Q&A] [answered]
def click_chain seem to perform only clicks, can't figure out how to call custom ActionChains

**Comment by mdmintz** [comment, marked as answer]
To perform a right-click / context-click, I would refer to this:
https://stackoverflow.com/questions/20316864/how-to-perform-right-click-using-selenium-chromedriver/20316987#20316987
To access the ``driver`` in SeleniumBase, use ``self.driver``.
